dataset_configs/english/librispeech/config.yaml

```diff
@@ -4,19 +4,21 @@ documentation: |
   
   This config can be used to prepare
   `Librispeech <https://openslr.org/12>`_
+  `Librispeech(mini) <https://openslr.org/31>`_
```

Comment:
Delete this line?

Reply:
Actually the thing is that mini dataset also consists of 2 subsets. This config allows you to download any subset of the librispeech dataset
  - ``"dev-clean"``
  - ``"dev-other"``
  - ``"test-clean"``
  - ``"test-other"``
  - ``"train-clean-100"``
  - ``"train-clean-360"``
  - ``"train-other-500"``
  - ``"dev-clean-2"``
  - ``"train-clean-5"``
  
  
  the last 2 are acually the subsets of mini dataset